src/pages/vendor/Vendor.tsx

```diff
@@ -12,7 +12,17 @@ import Header from "../../components/Header";
 import {loadSettings, saveSettings} from "../../utils/LocalStorage";
 import {VendorSettings} from "../../utils/SavedSettings";
 import {ProfileContext} from "../../components/profile/ProfileContext";
-
+import {gems} from "../../generated/GeneratedGems";
+import GemNameList from './GemNameList';
+
+// TODO: Ideally we'd pre-filter these out of GeneratedGems before generating
```

Comment:
Your call whether to do this before merging the PR or merge this and iterate after

Reply:
Agreed. This works for now tho